snc.sh

```diff
@@ -46,7 +47,7 @@ ${PODMAN} run --pull=always --rm -v ${PWD}/${CRC_INSTALL_DIR}/tmp/:/data:Z -w /d
 sudo mv ${CRC_INSTALL_DIR}/tmp/fedora-coreos-*-qemu.${ARCH}.qcow2 /var/lib/libvirt/images/fedora-coreos-qemu.${ARCH}.qcow2
 rmdir ${PWD}/${CRC_INSTALL_DIR}/tmp/
 
-sudo setfacl -m u:qemu:rx $HOME
+setfacl -m u:qemu:rx ${HOME}
```

Comment:
do we need both `setfacl` calls? One for `${HOME}` and another one for `${PWD}/${CRC_INSTALL_DIR}` ?

Reply:
No no, i was just testing things on the CI, but we should use the old command with recursive (`setfacl -m u:qemu:rx ${PWD}/${CRC_INSTALL_DIR}`) which works for both openshift CI and the downstream CI